documentation/book/ref-kafka-listener-authentication-example.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-broker-listener-network-policies.adoc
+
+[id='ref-kafka-listener-authentication-example-{context}']
+= Authentication configuration for a listener
+
+The following example shows a `plain` listener configured for SCRAM-SHA authentication and a `tls` listener with mutual TLS authentication
```

Comment:
there is no mention about the "external" one. We should remove it from the example or mentioning it in the description.

Reply:
Good spot. I think we can keep, so extended the intro to include the external listener